22-12-2021 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionError al cargar la clave del certificadoCodigo error afip:*****Numero de Error: 1429Linea de codigo de error: cms = WSAA.SignTRA(tra, cRuta + this.pcrt, cRuta + this.pkey) && Cert. DemoProcedimiento: sw_wsaa_iniciarError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              
.Exepción: RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              
Traceback: Traceback (most recent call last):
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              

22-12-2021 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionError al cargar la clave del certificadoCodigo error afip:*****Numero de Error: 1429Linea de codigo de error: cms = WSAA.SignTRA(tra, cRuta + this.pcrt, cRuta + this.pkey) && Cert. DemoProcedimiento: sw_wsaa_iniciarError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              
.Exepción: RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              
Traceback: Traceback (most recent call last):
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              

02-01-2023 19:30:05 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionError al cargar la clave del certificadoCodigo error afip:*****Numero de Error: 1429Linea de codigo de error: cms = WSAA.SignTRA(tra, cRuta + this.pcrt, cRuta + this.pkey) && Cert. DemoProcedimiento: sw_wsaa_iniciarError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       
.Exepción: RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       
Traceback: Traceback (most recent call last):
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       

02-01-2023 19:30:22 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionError al cargar la clave del certificadoCodigo error afip:*****Numero de Error: 1429Linea de codigo de error: cms = WSAA.SignTRA(tra, cRuta + this.pcrt, cRuta + this.pkey) && Cert. DemoProcedimiento: sw_wsaa_iniciarError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       
.Exepción: RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       
Traceback: Traceback (most recent call last):
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       

02-01-2023 19:30:47 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionError al cargar la clave del certificadoCodigo error afip:*****Numero de Error: 1429Linea de codigo de error: cms = WSAA.SignTRA(tra, cRuta + this.pcrt, cRuta + this.pkey) && Cert. DemoProcedimiento: sw_wsaa_iniciarError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       
.Exepción: RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       
Traceback: Traceback (most recent call last):
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.key                                       , J:\XSOFTSQL\PROYECTOS\HOTEL\AfipCert\hotelmu.crt                                       

